### DISCUSSION TITLE
Documentation for page size

### DISCUSSION THREAD

**Discussion by suestreet** [davidgohel/officer#559] [General]
Documentation for page size had me fooled
the example page_size(width = 21/2.54, height = 29.7/2.54, orient = "portrait") is standard A4 size in cm,
but the width / height units are inches. Thanks for all your work btw

**Comment by davidgohel** [comment]
Sorry, I did not understand your question.

The manual clearly states these arguments are expected to be expressed in "inches".

Can you explain what you are expecting or waht is the problem?

**Reply by suestreet** [reply]
Sorry, I was confused to see the A4 metric dimensions (210 x 297) and had misinterpreted the /2.54 component as meaning something different. It's not a problem, but maybe a comment that it's an example of how to load a metric size into the page size instruction would help the sadly confused newbies like me. It took a bit of experimentation to get the whole officedown + officer + flextable to work, but well worth the investment in being able to do some things that I really would like to be able to do. Thanks to the developers and maintainers
